packages/block-library/src/gallery/edit.js

```diff
@@ -686,6 +751,32 @@ export default function GalleryEdit( props ) {
 								/>
 							</ToolsPanelItem>
 						) }
+						{ aspectRatioOptions.length > 1 && (
+							<ToolsPanelItem
+								hasValue={ () =>
+									!! aspectRatio && aspectRatio !== 'auto'
+								}
+								label={ __( 'Aspect ratio' ) }
+								onDeselect={ () => setAspectRatio( 'auto' ) }
+								resetAllFilter={ () => ( {
+									aspectRatio: 'auto',
+								} ) }
+								isShownByDefault
+								panelId={ clientId }
```

Comment:
```suggestion
								isShownByDefault
```

Should work without `panelId`

Reply:
This also seems to cause resetting to not work correctly, lets remove it.